pom.xml

```diff
@@ -240,7 +240,7 @@
                         <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
                     </from>
                     <to>
-                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/lfp-pay-web:latest</image>
+                        <image>416670754337.dkr.ecr.eu-west-1.amazonaws.com/local/lfp-pay-web:latest</image>
```

Comment:
Please can this be `169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/lfp-pay-web:latest`
Looking at our aws profiles `~/.aws/config`, my understanding is that the image should belong to 
`profile development-eu-west-1` instead of `profile shared-services-ecr-eu-west-1-ro]`

Reply:
Updated from the suggestion above